README.md

```diff
@@ -196,10 +150,40 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
 
 **Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.1.0).
 
+---
+
+#### Android SDK: 4.8.0
+**Expected Release Date**: April 2, 2019
+
+**Contents**: Contents of this release can be tracked on the [Android 4.8.0 Project Page](https://github.com/smartdevicelink/sdl_android/projects/9).
+
+---
+
+#### iOS SDK: 6.2.0
+**Expected Release Date**: April 2, 2019
+
+**Contents**: Contents of this release can be tracked on the [iOS 6.2.0 Project Page](https://github.com/smartdevicelink/sdl_ios/projects/11).
+
+---
+
+#### SDL Protocol: 5.2.0
+**Expected Release Date**: April 2, 2019
+
+**Contents**: Contents of this release can be tracked on the [Protocol Spec 5.2.0 Project Page](https://github.com/smartdevicelink/protocol_spec/projects/3).
+
+---
+
+#### SDL RPC Spec: 5.1.0
+**Release Date**: April 2, 2019
+
+**Contents**: Contents of this release can be tracked on the [RPC Spec 5.1.0 Project Page](https://github.com/smartdevicelink/rpc_spec/projects/).
+
+---
+
+#### Core: 5.1.0
+**Expected Release Date**: April 2, 2019
 
+**Contents**: Contents of this release can be tracked on the [Core 5.1.0 Project Page](https://github.com/smartdevicelink/sdl_core/projects/13).
 
 ---
 
-[sdl_slack]: http://slack.smartdevicelink.com "SDL Slack"
-[sdl_evolution_channel]: https://smartdevicelink.slack.com/messages/sdl_evolution/ "sdl_evolution slack channel"
-[proposal-status]: https://smartdevicelink.github.io/sdl_evolution/
```

Comment:
Did you intend to remove these?

Reply:
I sure didn't.  I've added them back, thanks!